commands/view-state.js

```diff
@@ -36,8 +35,21 @@ async function viewState(options) {
     const { accountId, prefix, finality, blockId, utf8 } = options;
     const near = await connect(options);
     const account = await near.account(accountId);
-
-    let state = await account.viewState(prefix, { blockId, finality });
+    if (finality && blockId) {
+        console.error('Only one of --finality and --blockId can be provided');
+        process.exit(1);
+    } else if (!finality && !blockId) {
+        console.error('Must provide either --finality or --blockId');
```

Comment:
For 'we require either one of two options to be includes', use .check() (ref: https://yargs.js.org/docs/#api-reference-checkfn-globaltrue) and throw from the function if the validation fails (instead of process.exit())